tests/python_package_test/test_basic.py

```diff
@@ -430,6 +430,52 @@ def test_cegb_scaling_equalities(tmp_path):
         assert p1txt == p2txt
 
 
+def test_cegb_split_buffer_clean():
```

Comment:
Please move this test to `test_engine.py` file because `lgb.train()` function that is defined in `engine.py` is used in this tests.

Reply:
Resolved via a04d004